### ISSUE TITLE
[Feature Request] Toxic Rain DPS mode that includes duration

### ISSUE THREAD

**Issue by xanthics** [Openarl/PathOfBuilding#1179]
It would be nice to have a DPS calculation mode for Toxic Rain that takes duration and aps in to account for stacking pods.  Calculate the duration of a pod from initial hit to explosion.  During that time frame add initial hits and stacking dots per possible attack.  One pod per attack should be fine for calculating relative dps.

EG my current setup does 2.64 attacks per second with a duration of 1.05 seconds.  According to POB my average hit is 147.8 with a dot dps of 7189.4.  I'd like a field or setting that is the hit dps + the sum of all pods active during that second(pod duration * aps I think, which would be 2.772 pods or 19,929.0 dot DPS).  Would also affect "show node power".

TL;DR a Toxic Rain DPS mode for fully ramped damage that assumes 1 pod hits/dots per attack.
